core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -494,6 +497,20 @@ protected String webConsoleSchema() {
         return "http";
     }
 
+    public static final class AsyncMunmapJob extends SynchronizedJob {
+
+        private final MmapCache cache;
+
+        public AsyncMunmapJob() {
```

Comment:
have you considered a designated pool to do unmaps? Having synchronized job could quickly bog down the whole queue under heavy load, with 10k queries/s. Having unmaps on own thread pool seems like a win/win to me. These threads would either be sleeping because there is nothing to do or they will be sleeping because unmap bogged down.

Reply:
the job is no longer synchronized